src/common/k8s-api/kube-api.ts

```diff
@@ -34,6 +34,9 @@ import type { IKubeWatchEvent } from "./kube-watch-api";
 import { KubeJsonApi, KubeJsonApiData } from "./kube-json-api";
 import { noop } from "../utils";
 import type { RequestInit } from "node-fetch";
+
+// BUG: https://github.com/mysticatea/abort-controller/pull/22
```

Comment:
Could explain more about the relevancy of this bug?

Reply:
So there is the rule `import/no-named-as-default` from `eslint-import` which errors if (according to the typings) an default import uses the same name as a named non-default import because it might mean that you meant to import that instead.

However, in this case there is a bug in the "abort-controller" package where (even though according to the typings there is an export called "AbortController") in fact it is only default exported in the JS.